### ISSUE TITLE
Immediately affect of code change

### ISSUE THREAD

**Issue by ahrajabi** [python-telegram-bot/python-telegram-bot#381]
Hello,
After doing some changes on code, I must restart python for see results of changes. Is there any way to affect changes immediately?

**Comment by bomjacob** [comment]
Unfortunately this in not really in the scope of python. While there are several attempt at doing "code-hotswapping" in python. See for example [hotswap](https://pypi.python.org/pypi/hotswap/0.1) or [hottie](https://github.com/narfdotpl/hottie). But none of these would really work for this purpose, or at least not in a way that this library can support/provide - it's probably possible to do it on a case-by-case basis, but python isn't really made for this, so it's not recommended. 
**TLDR:** No, unfortunately not.
